app/views/layouts/_footer.html.erb

```diff
@@ -5,10 +5,10 @@
                 <h2 class="govuk-heading-m">Get help</h2>
 
                 <p class="govuk-body-s govuk-!-margin-bottom-1">
-                  Call <%= link_to("0800 389 2500", "tel://08003892500", class: "govuk-footer__link") %> or <%= link_to_git_site("chat online", "#talk-to-us", class: "govuk-footer__link") %>
+                <%= link_to("Call 0800 389 2500", "tel://08003892500", class: "govuk-footer__link") %> or <%= link_to_git_site("chat online", "#talk-to-us", class: "govuk-footer__link") %>
```

Comment:
```suggestion
                <%= link_to("Call the Get into Teaching team on 0800 389 2500", "tel://08003892500", class: "govuk-footer__link") %> or <%= link_to_git_site("chat online", "#talk-to-us", class: "govuk-footer__link") %>
```
I wonder if it makes more sense to say **who** you would be calling with this number @philspray - I think that is more what the DAC ticket is getting at? Although this is a bit of mouthful - you might be able to think of a better option!

(Otherwise all looks good!)